plugins/modules/incident.py

```diff
@@ -279,7 +282,7 @@ def run(module, table_client, attachment_client):
 
 def main():
     module_args = dict(
-        arguments.get_spec("instance", "sys_id", "number", "attachments"),
+        arguments.get_spec("instance", "sys_id", "number", "attachments", "mapping"),
```

Comment:
An example of mapping in the EXAMPLE section would help.

Reply:
Agreed - was allready there as doc_fragment. Inclusion is on line 34. 

It is doc fragment because it is used at lest at Incident and IncidentInfo.